cfme/tests/v2v/test_v2v_manual.py

```diff
@@ -0,0 +1,632 @@
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan with a VM with encrypted disk
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. Successful migration
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_vmware60_test_vm_with_multiple_disks():
+    """
+    OSP: vmware60-Test VM with multiple Disks
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        caseimportance: medium
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware60 to OSP
+            2. Create migration plan with a VM with multiple disk
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. Successful migration
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_vmware67_test_vm_with_multiple_disks():
+    """
+    OSP: vmware60-Test VM with multiple Disks
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        caseimportance: medium
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware60 to OSP
+            2. Create migration plan with a VM with multiple disk
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. Successful migration
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_policy_to_prevent_source_vm_from_starting_if_migration_is_complete():
+    """
+    OSP: Test policy to prevent source VM from starting if migration is
+    comAplete
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        caseimportance: medium
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware60 to OSP
+            2. Create migration plan with a VM with multiple disk
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. Successful migration
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_creating_multiple_mappings_with_same_name():
+    """
+    OSP: Test creating multiple mappings with same name
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        caseimportance: medium
+        caseposneg: negative
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping
+            2. Create infrastructure with duplicate name
+        expectedResults:
+            1.
+            2.
+            3. Duplicate name should not be allowed
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_migrations_with_multi_zonal_setup():
+    """
+    OSP: Test migrations with multi-zonal setup
```

Comment:
**REQUIRED**: Add missing `title` in this and all the below tests. Since some of the tests have it and others don't, let's make it consistent.